### PR TITLE
Sync telliot docs using Github actions

### PR DESCRIPTION
**Pull request by hhio618**
### Refs
[telliot#268](https://github.com/tellor-io/telliot/issues/268)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Sync miner docs from https://github.com/tellor-io/telliot/tree/master/docs in miner-documentation folder in the current repo
```
Comment (krasi-georgiev) [inline .github/workflows/main.yml:1]: We should rename the Miner folder to Telliot and will refactor the docs to explain that one of the tasks that Telliot does is mining.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      # Runs a set of commands using the runners shell
+      - name:  Sync
+        run: |
+          git clone https://github.com/tellor-io/telliot -b master
+          export TELLIOT_MD5=$(find ./telliot/docs  -type f -exec md5sum {} \; | md5sum | cut -d" " -f1)
+          export CURRENT_MD5=$(find $GITHUB_WORKSPACE/miner-documentation -type f -exec md5sum {} \; | md5sum | cut -d" " -f1)
+          if [ "$TELLIOT_MD5" != "$CURRENT_MD5" ];
+          then 
+            sed -i 's/(/(miner-documentation\//g' ./telliot/docs/INDEX.md
+            python .github/workflow/update-summary.py ./telliot/docs/INDEX.md
```
Comment (krasi-georgiev) [inline .github/workflows/main.yml:35]: Why is this needed?

Comment (hhio618) [reply .github/workflows/main.yml:35]: I put an INDEX.md file in Telliot docs folder which contains Telliot table of content. This way the main docs can be in sync with Telliot table of content and we don't need to manually edit the SUMMARY.md file every time the auto-sync process creates a PR.

Comment (krasi-georgiev) [reply .github/workflows/main.yml:35]: Aah yeah I see what you mean. Lets call it SUMMARY.md to be consistent and also when opening the Telliot dedicated gitbook space it will be used as well.

Comment (krasi-georgiev) [reply .github/workflows/main.yml:35]: Also in the Telliot repo we would need to add a `.gitbook.yaml`  to specify the root folder and the location of the summary there.

Comment (hhio618) [reply .github/workflows/main.yml]: Ok, then I changed the action to handle a SUMMARY.md file.

**Comment by krasi-georgiev** [comment]
I don't have merge access to this repo but asked for access and will merge soon

**Comment by krasi-georgiev** [comment]
Thanks!
